crates/dc_bundle/src/definition/view.rs

```diff
@@ -317,6 +317,8 @@ impl View {
         reactions: Option<Vec<Reaction>>,
         scroll_info: ScrollInfo,
         frame_extras: Option<FrameExtras>,
+        shader: Option<String>,
+        shader_fallback_color: Option<String>,
```

Comment:
We usually store colors as a `Color` struct, `Int`, or four `u8`s. Any reason this is a string?

Reply:
Not really. Using string is pretty straightforward but I realized that every time it draws it calls the parse color which is not very great so I changed it to use the Color struct.